FullyNoded/Helpers/Tor/MakeRPCCall.swift

```diff
@@ -183,4 +187,167 @@ class MakeRPCCall {
         }
         
     }
+    
+    func listUnspentUTXOs(completion: @escaping (Result<[UTXO], MakeRPCCallError>) -> Void) {
```

Comment:
same comment as above for this, we really out to have one dedicated class for them all instead of random places. As the logic flows now its: 

`Reducer.makeCommand` which handles common errors which then uses `MakeRPCCall.swift` to make generic commands

Both classes just handle whatever you pass it.

As I mentioned above I think what you are doing makes sense but best to put them all in a new (what will end up being huge) NodeCommand.swift class